ledger/src/blocktree.rs

```diff
@@ -157,7 +157,7 @@ impl Blocktree {
         adjust_ulimit_nofile();
 
         // Open the database
-        let measure = Measure::start("open");
+        let mut measure = Measure::start("open");
```

Comment:
fyi @mvines , I threw this in to make CI happy again

Reply:
:-/